oauth2server/proj/settings/default.py

```diff
@@ -81,4 +81,108 @@
         'rest_framework.authentication.BasicAuthentication',
         'rest_framework.authentication.SessionAuthentication',
     )
-}
\ No newline at end of file
+}
+
+
+# Custom logging to give us unique logging to work on remote containers and normal logging
+# Requirements for our custom loggers are:
+# 1)    A console logger that logs debug to console for DEBUG=TRUE and WARNINGS and higher when DEBUG=FALSE.
+# 2)    Disable all email sent to ADMINS via the mail_admins handler for the django.request and django.security loggers.
+# 3)    Add two custom logging formatters to enhance the logging output.
+# 4)    Add handlers to write files for 'proj' and 'app' name space. For production (DEBUG=FALSE) and dev (DEBUG=TRUE).
+# 5)    Add LOGGERS for 'apps' and 'proj' name space. Add LOGGERS for python and django default. Add a logger for remote
+#       files.
+
+LOGGING = {
```

Comment:
Console logs now look like this:

July 06, 2017 - 11:37:46
Django version 1.9.5, using settings 'proj.settings.local'
Starting development server at http://127.0.0.1:8000/
Quit the server with CONTROL-C.
 {"created":"2017-07-06 11:37:49 ", "service":"oauth2", "level": "DEBUG", "event":"Token validate request decorator being called", "context": "apps.tokens.decorators.validate_request:112"  }
 {"created":"2017-07-06 11:37:49 ", "service":"oauth2", "level": "DEBUG", "event":"Decorator called for Validating requests", "context": "apps.tokens.decorators.decorator:303"  }
 {"created":"2017-07-06 11:37:49 ", "service":"oauth2", "level": "DEBUG", "event":"Grant_type password", "context": "apps.tokens.decorators._validate_grant_type:155"  }
 {"created":"2017-07-06 11:37:49 ", "service":"oauth2", "level": "DEBUG", "event":"Grant_type password. Second function", "context": "apps.tokens.decorators._validate_grant_type:192"  }